packages/contracts-bedrock/test/L1/SystemConfigInterop.t.sol

```diff
@@ -91,13 +91,13 @@ contract SystemConfigInterop_Test is CommonTest {
             )
         );
 
-        vm.prank(systemConfig.owner());
+        vm.prank(_systemConfigInterop().dependencyManager());
```

Comment:
this is pranking as the zero address because `dependencyManager` is not set (similarly in the other unit tests doing this prank)

the `_cleanStorageAndInit` function is using the inherited initializer and not the new one in `SystemConfigInterop`.

Reply:
would be great to test that the `initialize` function in `SystemConfigInterop` sets the dependency manager address correctly

Reply:
would need to be done when `SystemConfig`and `SystemConfigInterop` are merged as one contract so that the old `initialize` does not have the `initializer` modifer and is made internal. I'll add a comment in the tests stating this so that its remembered.

A solution to add a test for it now however would be to remove the new `initialize` function's modifier so that calling it does not revert (it technically is the same thing as how it will be when deployed given that calling the `initialize` function is the first action the new `initialize` function takes)

Reply:
Would removing `initializer` from the new `initialize` function help here? And just rely on the internal `initialize` to have it?

Reply:
Yes, it won't revert in that case and makes writing a test for it possible